src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -13,34 +13,377 @@ import spray.http.StatusCodes
 /**
  * Created by dvoet on 2/4/16.
  */
-case class AttributeRecord(id: UUID,
+case class EntityAttributeRecord(id: Long,
```

Comment:
There's a lot of similar/repeated code in this AttributeComponent class. We have three types of attributes, each one differing slightly (entity attributes have a Long id to an entity, workspace attributes have a UUID to a workspace, and submission attributes have no foreign key to a parent)

Additionally, slick doesn't seem to provide a way to reuse code for different tables, so even if the data signatures matched up, we'd possibly still be stuck with duplicate code